FeedAPIChallenge/Feed API/FeedItemsMapper.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  FeedItemsMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Valeria Chub on 10.09.2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal final class FeedItemsMapper {
+	private static let STATUS_OK_200 = 200
+
+	internal static func map(data: Data, response: HTTPURLResponse) -> FeedLoader.Result {
+		guard
+			response.statusCode == STATUS_OK_200,
+			let root = try? JSONDecoder().decode(Root.self, from: data) else {
+			return .failure(RemoteFeedLoader.Error.invalidData)
+		}
+
+		return .success(root.feedImages)
+	}
+}
+
+extension FeedItemsMapper {
+	private struct Root: Decodable {
+		private let items: [Item]?
+
+		fileprivate var feedImages: [FeedImage] {
```

Comment:
Making this property `fileprivate` has no effect since `Root` is already private in the scope of this file
```suggestion
		var feedImages: [FeedImage] {
```